Xamarin.PropertyEditing.Mac/PropertyTableDelegate.cs

```diff
@@ -125,6 +125,14 @@ public override void ItemDidCollapse (NSNotification notification)
 				this.dataSource.DataContext.SetIsExpanded (group.Key, isExpanded: false);
 		}
 
+		public override nfloat GetRowHeight (NSOutlineView outlineView, NSObject item)
+		{
+			var facade = (NSObjectFacade)item;
+			var vm = facade.Target as PropertyViewModel;
+			var group = facade.Target as IGroupingList<string, PropertyViewModel>;
+			return  (group == null) ? vm.RowHeight : group.ElementAtOrDefault(1).RowHeight;
```

Comment:
If the group is empty, this will NRE. If there's cases of "this can't be null at this stage", don't use safeguard methods like `OrDefault`. Additionally, `IGroupingList` has an indexer, always prefer that over `ElementAt`:
```csharp
internal interface IGroupingList<TKey, TElement>
		: IGrouping<TKey, TElement>, IReadOnlyList<TElement>
```